### PR TITLE
Preparing for 1.2.0

### PR DESCRIPTION
**Pull request by Leon-Bichmann**
## PR checklist

- [x] This comment contains a description of changes (with reason)
- [x] `CHANGELOG.md` is updated
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] Documentation in `docs` is updated
- [x] If necessary, also make a PR on the [nf-core/diaproteomics branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/diaproteomics)

### CODE REVIEW

**Comment by Leon-Bichmann** [comment]
fixes #69

**Comment by lkuchenb** [comment]
The changes look generally good to me. Just one thing that I noticed in the context of the changes that affect parallelization: the workflow lacks resource descriptors for all processes. Currently, all processes are assigned the same resources, even those that do not support multithreading. In particular, many of the OpenMS TOPP tools (even though they offer a `-threads` flag) do not support multithreading.

File: README.md [hunk 1]
```diff
@@ -46,6 +46,15 @@ The workflow is based on the [OpenSwathWorkflow](http://openswath.org/en/latest/
 
 See [usage docs](https://nf-co.re/diaproteomics/usage) for all of the available options when running the pipeline.
 
+## Pipeline Summary
```
Comment (lkuchenb) [inline README.md]: Either add a pipeline summary or remove the NGS related dummy text

File: README.md [hunk 2]
```diff
@@ -54,13 +63,18 @@ The nf-core/diaproteomics pipeline comes with documentation about the pipeline:
 
 nf-core/diaproteomics was originally written by Leon Bichmann.
 
+We thank the following people for their extensive assistance in the development
```
Comment (lkuchenb) [inline README.md:57]: Add people or remove

File: nextflow_schema.json [hunk 3]
```diff
@@ -115,6 +115,10 @@
                     "help_text": "Example file:\nhttps://raw.githubusercontent.com/nf-core/test-datasets/diaproteomics/unimod.xml",
                     "description": "Path to unimod file needs to be provided",
                     "default": "https://raw.githubusercontent.com/nf-core/test-datasets/diaproteomics/unimod.xml"
+                },
+                "skip_dia_processing": {
+                    "type": "boolean",
+                    "description": "Set this flag if only you only want to generate spectral libraries from DDA data"
```
Comment (lkuchenb) [inline nextflow_schema.json]:
```suggestion
                    "description": "Set this flag if you only want to generate spectral libraries from DDA data"
```

**Review by lkuchenb — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Leon-Bichmann) [reply README.md:57]: done

File: README.md (hunk 1 above)
Comment (Leon-Bichmann) [reply README.md]: done

**Review by lkuchenb — APPROVED** [review]

**Pull request merged**